dawn/renderer/utils/utils.js

```diff
@@ -81,7 +81,14 @@ export const windowInfo = {
 
 export class Logger {
   constructor(processname, firstline) {
-    this.log_file = fs.createWriteStream(`./${Date.now()}-${processname}.log`, { flags: 'w' });
+    let path;
+    if (processname === 'dawn') {
+      path = require('electron').remote.app.getPath('desktop'); // eslint-disable-line global-require
```

Comment:
Main vs renderer difference. One throws an error if used in one process, the other throws an error if used in the other process.